tests/testLinearizeSquared.py

```diff
@@ -0,0 +1,179 @@
+import unittest
```

Comment:
There is far too much here that is duplicated from tests/testLinearizeLookupTable.py .  Please consolidate the code.